### PR TITLE
Fix `chewy:journal:clean` task for ruby 3.x

### PR DESCRIPTION
**Pull request by muk-ai**
This task seems to fails in ruby 3.x because keyword arguments are not expanded.

```
$ bundle exec rails "chewy:journal:clean"
rails aborted!
ArgumentError: wrong number of arguments (given 1, expected 0)

Tasks: TOP => chewy:journal:clean
(See full trace by running task with --trace)
```

ruby: 3.0.4
chewy: 7.2.7

I added double splat operator.

### CODE REVIEW

File: lib/tasks/chewy.rake [hunk 1]
```diff
@@ -96,7 +96,7 @@ namespace :chewy do
     task clean: :environment do |_task, args|
       delete_options = Chewy::RakeHelper.delete_by_query_options_from_env(ENV)
       Chewy::RakeHelper.journal_clean(
-        [
+        **[
           parse_journal_args(args.extras),
           {delete_by_query_options: delete_options}
         ].reduce({}, :merge)
```
Comment (muk-ai) [inline lib/tasks/chewy.rake:102]:
```
[1] pry(main)> [
[1] pry(main)*   parse_journal_args(args.extras),
[1] pry(main)*   {delete_by_query_options: delete_options}
[1] pry(main)* ].reduce({}, :merge)
=> {:only=>[], :delete_by_query_options=>{}}
```

**Comment by konalegi** [comment]
@muk-ai could you please provide a matrix for ruby 3.2 https://github.com/toptal/chewy/blob/master/.github/workflows/ruby.yml#L45?

**Comment by muk-ai** [comment]
My environment is ruby **3.0.4**, I don't think it has anything to do with ruby 3.2.

**Comment by konalegi** [comment]
ah, ok, right. Although do you think it's still possible to add kind of spec so your solution is tested on CI through all ruby versions that are defined there?

**Comment by muk-ai** [comment]
@konalegi I wasn't sure if I should add the test to this position, but I added it anyway.

**Comment by konalegi** [comment]
@muk-ai thanks for the tests, it looks good, could you please fix rubocop issues?

**Comment by muk-ai** [comment]
@konalegi Thanks for your quick review. fixed.

**Comment by konalegi** [comment]
sorry, I've forgot (shame on me), could you please provide an entry in the changelog, for example, https://github.com/toptal/chewy/pull/869/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR7-R9

**Comment by muk-ai** [comment]
@konalegi A changelog was added👍

**Review by konalegi — APPROVED** [review]

**Review by barthez — APPROVED** [review]

**Pull request merged**